manual/src/refman/patterns.etex

```diff
@@ -59,16 +59,42 @@ A pattern that consists in a value name matches any value,
 binding the name to the value. The pattern @"_"@ also matches
 any value, but does not bind any name.
 
+\begin{caml_example}{toplevel}
+let print_int_opt = function
+  | Some x -> print_int x
+  | _ -> ();;
```

Comment:
If someone were to write this code and send it for review, I would ask for `_` to be turned into `None` to be more precise and non-fragile.

Here would be an example where I would not suggest removing the underscore:

```ocaml
let is_empty = function
| [] -> true
| _::_ -> false
```